src/components/error/error-modal.component.tsx

```diff
@@ -19,7 +19,7 @@ const ErrorModal: React.FC<{ errors: ValidationResult[] }> = ({ errors }) => {
       }
     });
 
-    return Object.values(errorMessages).map((error: string, index) => <span key={index}>{error}</span>);
+    return Object.values(errorMessages).join(', ');
```

Comment:
The rationale for this change is that the [InlineNotification](https://react.carbondesignsystem.com/?path=/docs/components-notifications-inline--overview) component API no longer accepts a React Node as the value of the `subtitle` prop. So, this won't look great, but I'm not sure it ever did. Additionally, it looks like we're not using this component anywhere, so maybe we should remove it.